code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -87,6 +87,18 @@
 	origin_tech = "materials=2;magnets=2;programming=2;biotech=2"
 	var/id = 1
 
+
+/obj/item/weapon/implant/tracking/New()
+	..()
+	tracking_implants += src
+
+/obj/item/weapon/implant/tracking/Destroy()
+	..()
+	tracking_implants -= src
+
+
+var/global/tracking_implants = list()

```

Comment:
@Miauw62 YOU FUCK.
@Jalleo put the /global back.

```var/global/list/global_tracking_implants = list()```, and move it to the code/_globalvars/lists.dm file.